src/util/bundleUtils.ts

```diff
@@ -36,23 +36,42 @@ export async function createMeasurePackageBundle(measure: fhir4.Measure): Promis
     }
     const mainLib = libs[0];
 
-    const allLibsDups = await getAllDependentLibraries(mainLib as fhir4.Library);
```

Comment:
Smart thinking with refactoring this into its own function!

Reply:
Matt's idea!